packages/@sanity/base/src/module-status/types.ts

```diff
@@ -1,5 +1,19 @@
 import type {SanityClient} from '@sanity/client'
 
+export interface ChangeItem {
+  changeType: 'bugfix' | 'feature'
+  description: any
```

Comment:
We should probably type this, as it is quite hard to reason about the shape without knowing what is in it?

Rough untested sketch based on what I can tell from the shape:
```ts
import type {SanityImageHotspot, SanityImageCrop, SanityImageDimensions} from '@sanity/asset-utils'
import type {PortableTextBlock} from '@portabletext/types' // Remember to add this one as a dependency!

interface CodeBlock {
  _type: 'code'
  code?: string
  language?: string
  filename?: string
  highlightedLines?: number[]
}

interface MaterializedImage {
  _type: 'image'
  alt?: string
  caption?: string
  hotspot?: SanityImageHotspot
  crop?: SanityImageCrop
  asset?: {
    _id: string
    path: string
    metadata: {
      dimensions: SanityImageDimensions
      lqip?: string
    }
  }
}

type StudioBlockContent = 
  | PortableTextBlock
  | CodeBlock
  | MaterializedImage
```